apps/src/lib/levelbuilder/constants.js

```diff
@@ -1,4 +1,4 @@
 export const borderRadius = 3;
 export const tokenMargin = 7;
 
-export const PUBLISHED_STATES = ['pilot', 'beta', 'preview', 'recommended'];
+export const PUBLISHED_STATES = ['pilot', 'beta', 'preview', 'stable'];
```

Comment:
I would personally recommend turning this into an enum style constant, like this one: https://github.com/code-dot-org/code-dot-org/blob/0d7b7c7f6da9fee7c5e7f84c70b5171d77f2d8b3/apps/src/storage/dataBrowser/dataUtils.js#L3-L11  then you can use `PublishedState.STABLE` instead of `'stable'`. this makes the code more self-documenting, and allows IDEs and linters to catch typos.

Reply:
Thanks for the suggestion. Just updated

Reply:
I also added a new struct to SharedConstants.rb for the ruby code and updated that as well

Reply:
Updated to use the SharedConstants.rb struct everywhere